timemachine/fe/plots.py

```diff
@@ -441,6 +437,9 @@ def plot_hrex_replica_state_distribution_heatmap(
 
     fig.colorbar(p, label="fraction of iterations", format=lambda x, _: format_cbar_tick(x))
 
+    fig.suptitle(f"{prefix} HREX State Distribution Heatmap")
```

Comment:
nit: more precise, and to match function name:
```suggestion
    fig.suptitle(f"{prefix} HREX Replica-State Distribution Heatmap")
```